packages/neuron-ui/src/index.tsx

```diff
@@ -7,16 +7,18 @@ import 'styles/index.scss'
 import 'utils/i18n'
 import { useRoutes } from 'utils'
 
-import Navbar from 'containers/Navbar'
-import Notification from 'containers/Notification'
-import Main from 'containers/Main'
-import Settings from 'containers/Settings'
 import Transaction from 'components/Transaction'
 import SignAndVerify from 'components/SignAndVerify'
 import MultiSignAddress from 'components/MultisigAddress'
 import ErrorBoundary from 'components/ErrorBoundary'
+import Spinner from 'widgets/Spinner'
 import { withProvider } from 'states'
 
+const Navbar = lazy(() => import('containers/Navbar'))
+const Notification = lazy(() => import('containers/Notification'))
+const Main = lazy(() => import('containers/Main'))
```

Comment:
We can keep `Main` and `Navbar` eager, it's not native to show a loading page in a desktop app.